operate/client/src/App/ProcessInstance/BottomPanel/VariablePanel/tests/index.test.tsx

```diff
@@ -188,7 +187,8 @@ describe('VariablePanel', () => {
     expect(await screen.findByText('testVariableName')).toBeInTheDocument();
   });
 
-  (IS_LISTENERS_TAB_V2 ? it : it.skip)('should add new variable', async () => {
+  // FIXME: This test is broken when enabled
```

Comment:
@christoph-fricke I think we should fix these tests if we're enabling the feature

Reply:
@christoph-fricke Let's do a next one then but please create an issue so we don't lose track of it